test/libyul/ControlFlowSideEffectsTest.cpp

```diff
@@ -62,9 +62,9 @@ TestCase::TestResult ControlFlowSideEffectsTest::run(std::ostream& _stream, std:
 	if (!obj.hasCode())
 		BOOST_THROW_EXCEPTION(std::runtime_error("Parsing input failed."));
 
-	// TODO: Add EOF support
 	ControlFlowSideEffectsCollector sideEffects(
-		EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion(), std::nullopt),
+		EVMDialect::strictAssemblyForEVMObjects(solidity::test::CommonOptions::get().evmVersion(),
+			solidity::test::CommonOptions::get().eofVersion()),
```

Comment:
```suggestion
		EVMDialect::strictAssemblyForEVMObjects(
			CommonOptions::get().evmVersion(),
			CommonOptions::get().eofVersion()
		),
```

You don't need the `solidity::test` bit in most places already and you can just add `using namespace solidity::test` in those places that don't have it yet.